app/controllers/webhook_deliveries_controller.rb

```diff
@@ -0,0 +1,30 @@
+# WebhookDeliveriesController manages the updates of webhook deliveries.
+class WebhookDeliveriesController < ApplicationController
+  respond_to :html, :js
+
+  after_action :verify_authorized
+
+  # PATCH/PUT /namespaces/1/webhooks/1/deliveries/1
+  def update
```

Comment:
I said that I wanted you to move these (before) callbacks into the `update` method. I'd prefer to move the *code* altogether, not that you call them explicitly.  Thus, imo this method should be:

```ruby
def update
  @namespace = Namespace.find(params[:namespace_id])
  @webhook = @namespace.webhooks.find(params[:webhook_id])
  @webhook_delivery = @webhook.deliveries.find(params[:id])

  authorize @webhook_delivery

  @webhook_delivery.retrigger
  render template: "webhooks/retrigger", locals: { webhook_delivery: @webhook_delivery }
end
```

Moreover, it looks like some instance variables like `@namespace` and `@webhook` can be just mere locals. Am I right ?